graphql/types/OrderForm.graphql

```diff
@@ -115,6 +115,12 @@ type OrderFormItem {
   """
   unitMultiplier: Float
   canHaveAttachment: Boolean
+  attachments: [AttachmentItem]
```

Comment:
Can you change to this
```suggestion
  attachments: [Attachment!]!
```

https://github.com/vtex-apps/checkout-graphql/blob/master/graphql/types/Item.graphql#L14